TUM Campus App/CafeteriaViewController.swift

```diff
@@ -151,15 +151,15 @@ extension CafeteriaViewController {
 
 extension CafeteriaViewController: ASWeekSelectorViewDelegate {
     
-    func weekSelector(_ weekSelector: ASWeekSelectorView!, didSelect date: Date!) {
+    func weekSelector(_ weekSelector: ASWeekSelectorView, didSelect date: Date) {
```

Comment:
Have you tried these out? To check if they're getting called

Reply:
They are all getting called. I checked with breakpoints.